stable/elasticsearch-curator/templates/hooks/job.install.yaml

```diff
@@ -49,8 +49,9 @@ spec:
     {{- if $.Values.extraVolumeMounts }}
 {{ toYaml $.Values.extraVolumeMounts | indent 12 }}
     {{- end }}
-          command: [ "curator" ]
-          args: [ "--config", "/etc/es-curator/config.yml", "/etc/es-curator/action_file.yml" ]
+          command:
+{{ toYaml $.Values.command | indent 12 }}
+          args: [ "--dry-run", "--config", "/etc/es-curator/config.yml", "/etc/es-curator/action_file.yml" ]
```

Comment:
Why dry-run?

Reply:
so we don't actually apply the config on a helm post-install/upgrade hook, rather we check if config is runnable. Isn't this the intention?

Reply:
AFAIK, this is not a test but a way to run curator after install/upgrade.
Indeed, having a test that runs the curator with dry-run after install and upgrade to validate our configuration would be nice! But that should go in another file.

Reply:
and as far as I can interpret from:

> Add helm hooks to run jobs on install or upgrade.

`jobs` might mean k8s jobs or ES jobs, it wasn't very specific :/

If the intention is not to be a test, then yes it makes sense what you suggest.
If you believe we need both use-cases, then we can do it.

Reply:
You could add a separate test to check the config, but as a user I expect the job to run job when I enable the hooks, unless I have `.Values.dryrun: true` set.

Reply:
I've added that option, and an extra one at hook-level dry-run, please let me know your thoughts